javascript/javascript3/week1/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="week1.js"></script>
```

Comment:
Your app is working fine, the only problem is the position of your script tag. It very important to place your JavaScript tag or file in the bottom of the body tag . Something it cause problem when it is locate in the header.